packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryResetPassword/index.tsx

```diff
@@ -236,7 +236,12 @@ const AccountRecoveryResetPassword = ({
           break;
         case IntegrationType.OAuth:
           // TODO just use type guard instead of switch, FXA-8111
-          if (sessionIsVerified && isOAuthIntegration(integration)) {
+          if (
```

Comment:
This also might not be needed and may conflict after https://github.com/mozilla/fxa/pull/15820 lands, since that PR means we aren't redirecting.